setup.py

```diff
@@ -80,8 +80,8 @@
                       'requests>=2.18.1',
                       'lxml>=4.2.1',
                       'pygithub>=1.51',
-                      'altair',
-                      'scipy>=1.4.0'
+                      'altair>=4.1.0',
```

Comment:
curious where you got the specific version number from

Reply:
It was what I had installed. We can certainly change this if you'd like. We'd need to identify the earliest working version possibly.

Reply:
Mine works on 4.0.1. @andersonfrailey -- you authored the initial PR for this, do you know what the VN is?

Reply:
I played around locally with hard coding the version. It seems the earliest one that doesn't crash on the examples is 4.0.0

I can update to that if you'd like.

Reply:
sounds good